onnx/defs/nn/defs.cc

```diff
@@ -1077,6 +1086,11 @@ ONNX_OPERATOR_SET_SCHEMA(
             {"tensor(float16)", "tensor(float)", "tensor(double)"},
             "Constrain input and output types to float tensors.")
         .SetDoc(LpNormalization_ver1_doc)
+        .Attr(
+            "epsilon",
+            "The epsilon value to use to avoid division by zero, default is 1e-10f.",
+            AttributeProto::FLOAT,
+            1e-10f)
```

Comment:
For the single floating number, 1e-10f is too small. Let's make it 1e-5.

Reply:
Tensorflow uses 1e-12 - https://www.tensorflow.org/api_docs/python/tf/nn/l2_normalize 
and MXNet uses 1e-10 - https://mxnet.incubator.apache.org/api/python/symbol/symbol.html#mxnet.symbol.L2Normalization. 

IMO 1e-10 is a pretty reasonable default value to have. Let me know if you still want the default value changed.

Reply:
I checked with the floating converter, it is fine. Let's keep 1e-10.